sdks/java/core/src/main/java/org/apache/beam/sdk/coders/Coder.java

```diff
@@ -198,7 +198,8 @@ public static void verifyDeterministic(Coder<?> target, String message, Iterable
     }
   }
 
-  public static <T> long getEncodedElementByteSizeUsingCoder(Coder<T> target, T value) throws Exception {
+  public static <T> long getEncodedElementByteSizeUsingCoder(Coder<T> target, T value)
+      throws Exception {
```

Comment:
FYI this keeps getting auto-formatted when running locally, so just leaving it instead of reverting all the time. Looks like there was a recent change that merged without the auto-formatting.